it/helpers/TaxYearHelper.scala

```diff
@@ -14,15 +14,14 @@
  * limitations under the License.
  */
 
-package utils
-import org.joda.time.{DateTime, DateTimeZone}
+package helpers
 
-object IntegrationTestClock extends Clock {
-  private val year = 2021
-  private val month = 1
-  private val day = 1
-  private val hour = 0
-  private val minute = 0
+import org.joda.time.DateTime
 
-  override def now(zone: DateTimeZone = DateTimeZone.UTC): DateTime = new DateTime(year, month, day, hour, minute, zone)
+object TaxYearHelper {
```

Comment:
It is better if this class is in the test (instead of it) so that we can use it for unit tests too.